packages/admin-panel/src/utilities/createNestedReducer.js

```diff
@@ -29,5 +29,5 @@ export const createNestedReducer = (defaultNestedState, nestedStateChanges) => {
   Object.entries(nestedStateChanges).forEach(([actionType, actionHandler]) => {
     stateChanges[actionType] = createNestedStateChanger(defaultNestedState, actionHandler);
   });
-  return createReducer({}, stateChanges);
+  return createReducer(defaultNestedState, stateChanges);
```

Comment:
This fixes an existing bug where the default state of the reducers was net setup for createdNestedReducer types.